db/kvstore4trie.go

```diff
@@ -94,12 +94,15 @@ func (s *KVStoreForTrie) Put(key []byte, value []byte) error {
 func (s *KVStoreForTrie) Get(key []byte) ([]byte, error) {
 	trieKeystoreMtc.WithLabelValues("get").Inc()
 	v, err := s.cb.Get(s.bucket, key)
-	if err != nil {
-		if v, err = s.dao.Get(s.bucket, key); err != nil {
-			return nil, errors.Wrapf(err, "failed to get key %x", key)
+	if errors.Cause(err) == ErrNotExist {
+		if v, err = s.dao.Get(s.bucket, key); errors.Cause(err) == ErrNotExist {
```

Comment:
feel line 98~99 can simply reduce to:
v, err = s.dao.Get(s.bucket, key)
there's no need for another wrap